File: Modul3/Python/PRAK304-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
i = 1

while i <= 5:
    a = int(input())
    
    if 20 <= a <= 99 or a == 10:
        print("Puluhan")
    elif 11 <= a <= 19:
        print("Belasan")
    elif 1 <= a <= 9:
        print("Satuan")
    elif a >= 100 or a < 0:
        print("Anda Menginput Melebihi Limit Bilangan")
    else:
        print("Nol")
    
    i += 1

File: Modul3/C/PRAK304-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>

int main(){

    int a;
    int i = 1;

    while (i <= 5){
        (scanf("%d", &a));{
            if (a >= 20 && a <= 99 || a == 10 ){
                printf("Puluhan\n");
                i++;
            }else if ( a >= 11 && a < 20){
                printf("Belasan\n");
                i++;
            }else if (a > 0 && a <= 9){
                printf("Satuan\n");
                i++;
            }else if (a >= 100 || a < 0){
                printf("Anda Menginput Melebihi Limit Bilangan\n");
                i++;
            }else{
                printf("Nol\n");
                i++;
            }  
        } 
    }
}